checks/evaluation/dangerous_workflow.go

```diff
@@ -30,6 +30,10 @@ func DangerousWorkflow(name string, dl checker.DetailLogger,
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
+	if r.NumWorkflows == 0 {
```

Comment:
Use `len(Workflows)`? Do we need this extra field in the struct?

Reply:
This is intentional, `Workflows` doesn't have all workflows, just ones we've found with risky security patterns.